### ISSUE TITLE
Missing banner on old WordCamp sites

### ISSUE THREAD

**Issue by outdoor2kode** [WordPress/wordcamp.org#1338]
### Describe the bug

There should be a banner on an old WC site that directs users to a current WordCamps site.
Related: https://github.com/WordPress/wordcamp.org/issues/1258
Design enhancement suggestions: 
https://wordpress.slack.com/archives/C08M59V3P/p1717079064398609?thread_ts=1717000539.205949&cid=C08M59V3P
https://wordpress.slack.com/archives/C08M59V3P/p1717079262338759?thread_ts=1717000539.205949&cid=C08M59V3P
https://wordpress.slack.com/archives/C08M59V3P/p1717079406356639?thread_ts=1717000539.205949&cid=C08M59V3P
cc @WordPress/meta-design 

### To reproduce

1. Go to https://us.wordcamp.org/2023/

### Expected behavior

There should be a banner on the top, just like [WCAsia site](https://asia.wordcamp.org/2023/).

Note: It appears not all WordCamp sites have this issue.

### Screenshots / Screencasts

**US**
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/8a57dfad-72a3-459c-8db4-2dcc757fa927)
**Asia**
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/f7652a38-3cf3-4d94-b673-386f8ab4607a)

**Comment by jasmussen** [comment]
Let's add the banner for sure. From the conversation it appears there are some technical, and verbiage suggestions, I'll defer to those. There's also a suggestion to make the banner bigger. We can, though I wouldn't make it quite as big as the suggestion, I'd think it should be no higher than the main navigation banner. It's important to remember that although the upcoming/next/most recent WordCamp is important to point to, there are legitimate archival reasons for visiting the old ones, including rewatching talks you liked, adding photos, etc.

The WCAsia banner can work as a design. Alternately, an inverted version that's identical to what's currently on the wp.org homepage, would be my recommendation.

**Comment by kcristiano** [comment]
+1  having a call to action so a user can go to the current site should be automated.  

> Note: It appears not all WordCamp sites have this issue.

I also see that https://europe.wordcamp.org/2023/  and https://europe.wordcamp.org/2022/ lack the banner.   Does the banner currently need to be added manualy?

**Comment by ryelle** [comment]
>  Does the banner currently need to be added manualy?

No, it was disabled for sites that have future sites already created (see #1258), because those were redirecting to the future event. Currently, it's automatically on every camp, _except_ some we've manually disabled it on (setting a feature flag from a sandbox #1260). It was also manually turned off for [an alternate language site (WC Canada fr)](https://wordpress.slack.com/archives/C08M59V3P/p1709292967283289?thread_ts=1706836174.954409&cid=C08M59V3P).

See also #1239 for another attempted approach.

**Comment by timiwahalahti** [comment]
I think #1258 could be avoided by checking the site's corresponding WordCamp Post on Central and if the end date is in the past. That would probably solve edge cases like this. Tho now that the feature flag is implemented, I'd keep it for quick situations that might arise.
